common/v2/config/networks/globalProvider.ts

```diff
@@ -0,0 +1,81 @@
+import { ethers } from 'ethers';
+import { BaseProvider } from 'ethers/providers';
+import { PROVIDER_OPTIONS } from './providerOptions';
+import { Network } from 'v2/services/Network/types';
+
+export type NetworkKey = keyof typeof PROVIDER_OPTIONS;
+
+type FallbackProvider = ethers.providers.FallbackProvider;
+const FallbackProvider = ethers.providers.FallbackProvider;
+
+type TempProviders = { [K in NetworkKey]: BaseProvider[] };
+type FallbackProviders = { [K in NetworkKey]: FallbackProvider };
+
+// tslint:disable-next-line
+function entries<T, K extends keyof T>(obj: { [K in keyof T]: T[K] }): [K, T[K]][] {
+  return Object.entries(obj) as any;
+}
+
+function createFallBackProvidersFrom(config: typeof PROVIDER_OPTIONS): FallbackProviders {
```

Comment:
@vilsbole Ethers.js has it's own handling of etherscan because they have a different format. So can't provider a default etherscan default, we just tell Ethers.js that this node is etherscan, and it'll create the provider.